src/main.cpp

```diff
@@ -78,15 +154,44 @@ void loop(){
   // Waveforms can be combined, to create new wavefroms, see driver datasheet
   driver.setWaveform(0, effect);  // Setup the waveform(s)
   driver.setWaveform(1, 0);       // end of waveform waveform
+
   // When button is pressed, vibrate
-  if(triggerPress == "PUSH"){
+  if (triggerPress == "PUSH")
+  {
     driver.go();
   }
 
+  if (received_gait_instruction_response)
+  {
+    // This means gait instruction handled
+    triggerPress = "EXIT_GAIT";
+    received_gait_instruction_response = false;
+  }
```

Comment:
Why do you exit the gait once you receive a response?

Reply:
To continue to the next screen on the ipd, this allows the pilot to go to the next gait already